Homework_1/Task_1.py

```diff
@@ -0,0 +1,4 @@
+user_name = input("enter your name: ")
+User_lastname = input("enter your last name: ")
+user_age = input("enter how old are you: ")
+print("გამარჯობა,",user_name,User_lastname,"თქვენ ხართ",user_age,"წლის")
```

Comment:
გამოიყენეთ ფორმატერი და დაიცავით სტილი. 
```suggestion
print("გამარჯობა,", user_name, User_lastname, "თქვენ ხართ", user_age, "წლის")
```